### PR TITLE
Add CHANGELOG for 1.19.0

### PR DESCRIPTION
**Pull request by delvh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,359 @@ This changelog goes through all the changes that have been made in each release
...
+  * Add Chef package registry (#22554)
+  * Support asciicast files as new markup (#22448)
+  * cgo cross-compile for freebsd (#22397)
+  * Add cron method to gc LFS MetaObjects (#22385)
+  * Add new captcha: cloudflare turnstile (#22369)
+  * Enable `@<user>`- completion popup on the release description textarea (#22359)
+  * make /{username}.png redirect to user/org avatar (#22356)
+  * Add Conda package registry (#22262)
+  * Support org/user level projects (#22235)
+  * Add Mermaid copy button, avoid unnecessary tooltip hide (#22225)
```
Comment (silverwind) [inline CHANGELOG.md]:
```suggestion
  * Add Mermaid copy button (#22225)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,359 @@ This changelog goes through all the changes that have been made in each release
...
+  * Purge API comment (#23451) (#23452)
+  * User creation API: allow custom "created" timestamps (#22549)
+  * Add `updated_at` field to PullReview API object (#21812)
+  * Add API management for issue/pull and comment attachments (#21783)
+  * Add API endpoint to get latest release (#21267)
+  * Support system hook API (#14537)
+* ENHANCEMENTS
+  * Add `.patch` to `attachment.ALLOWED_TYPES` (#23580) (#23582)
+  * Fix sticky header in diff view (#23554) (#23568)
+  * Fix some broken css (#23560) (#23567)
```
Comment (silverwind) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,359 @@ This changelog goes through all the changes that have been made in each release
...
+  * Preserve file size when creating attachments (#23406) (#23426)
+  * Fix broken Chroma CSS styles (#23174) (#23402)
+  * Fix incorrect NotFound conditions in org/projects.go (#23384) (#23395)
+  * Set `X-Gitea-Debug` header once (#23361) (#23381)
+  * Pass context to avatar for projects view (#23359) (#23378)
+  * Fix panic when getting notes by ref (#23372) (#23377)
+  * Do not recognize text files as audio (#23355) (#23368)
+  * Fix adding of empty class name (#23352) (#23360)
+  * Fix various ImageDiff/SVG bugs (#23312) (#23358)
+  *  Fix incorrect display for comment context menu (#23343) (#23344)
```
Comment (silverwind) [inline CHANGELOG.md]:
```suggestion
  * Fix incorrect display for comment context menu (#23343) (#23344)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,359 @@ This changelog goes through all the changes that have been made in each release
...
+  * Make `gitea serv` respect git binary home (#23138) (#23197)
+  * Change button text for commenting and closing an issue at the same time (#23135) (#23182)
+  * Fix DBConsistency checks on MSSQL (#23132) (#23134)
+  * Show empty repos in Admin Repository Management page (#23114) (#23130)
+  * Redirect to the commit page after applying patch (#23056) (#23127)
+  * Fix nil context in RenderMarkdownToHtml (#23092) (#23108)
+  * Make issue meta dropdown support Enter, confirm before reloading (#23014) (#23102)
+  * Fix SyncOnCommit always return false in API of push_mirrors (#23088) (#23100)
+  * Fix commit name in Apply Patch page (#23086) (#23099)
+  *  Fix some more hidden problems (#23074) (#23075)
```
Comment (silverwind) [inline CHANGELOG.md]:
```suggestion
  * Fix some more hidden problems (#23074) (#23075)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,359 @@ This changelog goes through all the changes that have been made in each release
...
+  * Bump golang.org/x/net from 0.4.0 to 0.7.0 (#22980)
+  * Get rules by id when editing branch protection rule (#22932)
+  * Fix panic when call api (/repos/{owner}/{repo}/pulls/{index}/files) (#22921)
+  * Increase Content field size of gpg_import_key to MEDIUMTEXT (#22897)
+  * Fix hidden commit status on multiple checks (#22889)
+  * Fix .golangci.yml (#22868)
```
Comment (silverwind) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,359 @@ This changelog goes through all the changes that have been made in each release
...
+  * Use "Title Case" for text "Reference in new issue" (#22936)
+* BUILD
+  * Wrap unless-check in docker manifests (#23079) (#23081)
+  * Adjust manifest to prevent tagging latest on rcs (#22811)
+  * update to build with go1.20 (#22732)
+  * Add Bash and Zsh completion scripts (#22646)
+  * Add Contributed backport command (#22643)
+  * Remove deprecated packages & staticcheck fixes (#22012)
+  * Update to Alpine 3.17 (#21904)
+  * Update JS dependencies (#21881)
```
Comment (silverwind) [inline CHANGELOG.md]:
```suggestion
```
Another entry in misc, don't need it twice

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,359 @@ This changelog goes through all the changes that have been made in each release
...
+  * docs: add swagger.json file location to FAQ (#22489)
+  * Update index.de-de.md (#22363)
+  * Update Gmail mailer configuration (#22291)
+  * Add missed reverse proxy authentication documentation (#22250)
+  * Add plural definitions for German translations (#21802)
+  * Attempt clarify AppWorkPath etc. (#21656)
+  * Add some documentation to packages (#21648)
+* MISC
+  * Use `<nav>` instead of `<div>` in the global navbar (#23125) (#23533)
+  * Bump webpack from 5.75.0 to 5.76.0 (#23484) (#23487)
```
Comment (silverwind) [inline CHANGELOG.md]:
```suggestion
```

Not really notable

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,355 @@ This changelog goes through all the changes that have been made in each release
...
+  * Split migrations folder (#21549)
+  * feat: add button to quickly clear merge message (#21548)
+  * Add `context.Context` to more methods (#21546)
+  * Add index for hook_task table (#21545)
+  * Allow disable code tab (#20805)
+* BUGFIXES
+  * Fix diff detail buttons wrapping, use tippy for review box (#23271) (#23546)
+  * Handle missing `README` in create repos API (#23387) (#23510)
+  * Disable sending email after push a commit to a closed PR (#23462) (#23492)
+  *  Fix aria.js bugs: incorrect role element problem, mobile focus problem, tippy problem (#23450) (#23486)
```
Comment (silverwind) [inline CHANGELOG.md]:
```suggestion
  * Fix aria.js bugs: incorrect role element problem, mobile focus problem, tippy problem (#23450) (#23486)
```

It also fails lint 😆 

```
CHANGELOG.md:240:1 MD030/list-marker-space Spaces after list markers [Expected: 1; Actual: 2]
```

**Review by silverwind — APPROVED** [review]

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,355 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.19.0](https://github.com/go-gitea/gitea/releases/tag/1.19.0) - 2023-03-19
+
+* BREAKING
+  * Add loading yaml label template files (#22976) (#23232)
+  * Make issue and code search support camel case (#22829)
```
Comment (lafriks) [inline CHANGELOG.md]:
```suggestion
  * Make issue and code search support camel case for Bleve (#22829)
```

**Review by lafriks — CHANGES_REQUESTED** [review]

**Review by lafriks — APPROVED** [review]

File: CHANGELOG.md [hunk 10]
```diff
@@ -4,6 +4,355 @@ This changelog goes through all the changes that have been made in each release
...
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.19.0](https://github.com/go-gitea/gitea/releases/tag/1.19.0) - 2023-03-19
+
+* BREAKING
+  * Add loading yaml label template files (#22976) (#23232)
+  * Make issue and code search support camel case for Bleve (#22829)
+  * Repositories: by default disable all units except code and pulls on forks (#22541)
+  * Support template for merge message description (#22248)
+  * Fix wrong default value for update checker on app.example.ini (#22084)
```
Comment (wxiaoguang) [inline CHANGELOG.md]: It's not breaking, it's just a document change ....

File: CHANGELOG.md [hunk 11]
```diff
@@ -4,6 +4,355 @@ This changelog goes through all the changes that have been made in each release
...
+  * Remove ONLY_SHOW_RELEVANT_REPOS setting (#21962)
+  * Implement actions (#21937)
```
Comment (wxiaoguang) [inline CHANGELOG.md:15]: Why it's a breaking?

Comment (lunny) [reply CHANGELOG.md:15]: Added a new reserved user name `gitea-actions`.

Comment (wxiaoguang) [reply CHANGELOG.md:15]: But it is a big feature indeed. the "reserved user name" is just a side effect. 

Such unclear log doesn't tell end users any valuable information.

**Comment by lunny** [comment]
wait #23577

File: CHANGELOG.md (hunk 10 above)
Comment (techknowlogick) [reply CHANGELOG.md]:
```suggestion
```

**Pull request merged**